UnattendResources/Logon.ps1

```diff
@@ -376,6 +376,8 @@ try {
     $enableAdministrator = Get-IniFileValue -Path $configIniPath -Section "DEFAULT" `
                                             -Key "enable_administrator_account" -Default $false -AsBoolean
     $goldImage = Get-IniFileValue -Path $configIniPath -Section "DEFAULT" -Key "gold_image" -Default $false -AsBoolean
+    $disableFirstLogonAnimation = Get-IniFileValue -Path $configIniPath -Section "DEFAULT" -Key "disable_first_logon_animation" `

```

Comment:
add this after useIpv6EUI64 and also needs to have a try /catch because ini modules fails if that key is not found (for backward compat)